srv/components/ha/corosync-pacemaker/config/cluster_ip.sls

```diff
@@ -40,6 +40,11 @@ Setup ClusterIP resouce:
   cmd.run:
     - name: pcs resource create ClusterIP ocf:heartbeat:IPaddr2 ip={{ pillar['cluster']['cluster_ip'] }} mac=$(echo "01:00:5e:`echo {{ grains['hwaddr_interfaces'][data_if] }} | cut -d ":" -f 1-3`") nic={{ data_if }} op monitor interval=30s
 
+Update ClusterIP:
+  cmd.run:
+    - name: pcs resource update ClusterIP cidr_netmask=$(echo `ip addr show { data_if } | grep "inet\b" | awk '{print $2}' | cut -d "/" -f2` | awk '{print $1;})
```

Comment:
There is a small issue in `cidr_netmask` command, you lost the closing `'` symbol in the last `awk` command

Reply:
Also, for example, command can be simplified to
```
ip addr show enp179s0f0 | grep -m 1 "inet\b" | awk '{print $2}' | cut -d "/" -f2
```